unity/acsShowcase/Assets/Scripts/Calling/HandleIncomingCall.cs

```diff
@@ -68,7 +68,14 @@ public void Accept()
                 };
 
                 await HangUpCurrentCall();
-                CurrentCall = await acceptCall.AcceptAsync(acceptCallOptions);
+                if (acceptCall is TeamsIncomingCall teamsCall)
+                {
+                    CurrentCall = await teamsCall.AcceptAsync(acceptCallOptions);
+                }
+                else if (acceptCall is IncomingCall acsCall)
+                {
+                    CurrentCall = await acsCall.AcceptAsync(acceptCallOptions);
+                }
```

Comment:
add an else if with an error 

`Log.Error<HandleIncomingCall>($"Unable to accept call. Unknown call type {typeof(acceptCall)}.");`